public/data/kumaran.json

```
@@ -0,0 +1,27 @@
+{
+    "name": "T.kumaran",
+    "type": "Cybersecurity Enthusiast",
```

I think this should be `personal`. 😅 
```suggestion
    "type": "personal",
```